src/main/java/baseball/GameController.java

```diff
@@ -0,0 +1,67 @@
+package baseball;
+
+import static baseball.message.SystemMessage.*;
+
+import baseball.domain.Computer;
+import baseball.domain.Player;
+import baseball.domain.Referee;
+
+import camp.nextstep.edu.missionutils.Console;
+
+public class GameController {
+	public static final String RESTART_COMMAND = "1";
+	public static final String END_COMMAND = "2";
+
+	private final Player player;
+	private final Computer computer;
+	private final Referee referee;
+
+	public GameController() {
+		player = new Player();
+		computer = new Computer();
+		referee = new Referee();
+	}
+
+	public void run() {
+		do {
+			readyForGame();
+			startGame();
+		} while (restartGame());
+	}
+
+	private void readyForGame() {
+		computer.generateComputerNumber();
+		referee.saveComputerNumber(computer.getComputerNumber());
+	}
+
+	private void startGame() {
+		do {
+			inputPlayerNumber();
+			referee.judgeResult( player.getPlayerNumber() );
+			System.out.println( referee.getResult() );
+		} while ( !referee.isAnswer() );
```

Comment:
```suggestion
		} while (!referee.isAnswer());
```

코드 전반적으로 공백과 관련된 컨벤션이 뒤죽박죽입니다!
자동으로 코드 컨벤션에 맞게 포맷팅해주는 기능을 IDE에서 제공 중이에요.

https://xlffm3.github.io/ide/intellij_save_actions/

Reply:
save actions의 경우 프로젝트마다 설정을 해줘야하나요?? 
다른 프로젝트로 들어가보니 체크가 다 해제되어 있네요 ㅎㅎ 
또한, 파일 저장이 맥북 기준 cmd + s를 뜻하는 게 맞을까요??
좋은 정보 감사합니다!!